### PR TITLE
schema: extracting some missed gloss

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR adds/extracts some previously missed gloss entries.

### CODE REVIEW

File: source/modules/MEI.critapp.xml [hunk 1]
```diff
@@ -64,7 +64,8 @@
     </classes>
   </classSpec>
   <elementSpec ident="app" module="MEI.critapp">
-    <desc xml:lang="en">(apparatus) – Contains one or more alternative encodings.</desc>
+    <gloss versionDate="2023-06-16" xml:lang="en">apparatus entry</gloss>
```
Comment (musicEnfanthen) [inline source/modules/MEI.critapp.xml]:
```suggestion
    <gloss versionDate="2023-06-16" xml:lang="en">apparatus</gloss>
```

File: source/modules/MEI.cmn.xml [hunk 2]
```diff
@@ -1645,6 +1645,7 @@
     </remarks>
   </elementSpec>
   <elementSpec ident="graceGrp" module="MEI.cmn">
+    <gloss versionDate="2023-06-16" xml:lang="en">grace note group</gloss>
```
Comment (musicEnfanthen) [inline source/modules/MEI.cmn.xml]: grace group?

File: source/modules/MEI.critapp.xml (hunk 1 above)
Comment (rettinghaus) [reply source/modules/MEI.critapp.xml]: Took this from the original in TEI.

File: source/modules/MEI.cmn.xml (hunk 2 above)
Comment (rettinghaus) [reply source/modules/MEI.cmn.xml]: "grace group" sound odd to me.

Comment (pe-ro) [reply source/modules/MEI.cmn.xml]: Change to "Group of grace notes"?

Comment (ahankinson) [inline source/modules/MEI.cmn.xml]: As it's a glossary entry I think it's just supposed to be expanding any abbreviated or shortened components of the element names. So "graceGrp" == "Grace Group".

```suggestion
    <gloss versionDate="2023-06-16" xml:lang="en">grace group</gloss>
```

Comment (ahankinson) [reply source/modules/MEI.cmn.xml]: (See also `fingGrp` below...)

File: source/modules/MEI.neumes.xml [hunk 3]
```diff
@@ -378,6 +378,7 @@
     </attList>
   </elementSpec>
   <elementSpec ident="ncGrp" module="MEI.neumes">
+    <gloss versionDate="2023-06-16" xml:lang="en">neume components group</gloss>
```
Comment (ahankinson) [inline source/modules/MEI.neumes.xml]:
```suggestion
    <gloss versionDate="2023-06-16" xml:lang="en">neume component group</gloss>
```

File: source/modules/MEI.critapp.xml (hunk 1 above)
Comment (musicEnfanthen) [reply source/modules/MEI.critapp.xml]: @rettinghaus Can you resolve this? There seems to be a preference to go with just "apparatus" for now.

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Thx @rettinghaus, looks good to me.

**Review by bwbohl — APPROVED** [review]

**Pull request merged**